packages/manager/modules/pci/src/projects/project/ai-dashboard/ai-dashboard.component.js

```diff
@@ -0,0 +1,19 @@
+import controller from './ai-dashboard.controller';
+import template from './ai-dashboard.html';
+
+import './style.scss';
```

Comment:
**nitpick:** I dont see a `style.css` file in the folder. Could you please remove it?